### PR TITLE
Update H/W prereq

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/prereq-hardware.adoc [hunk 1]
```diff
@@ -90,52 +95,24 @@ sudo systemctl start var-lib-gravity-planet-etcd.mount
 
 == Docker Device
 
-Unless specified, Docker configuration defaults to the use of Device Mapper in loopback mode (using `/dev/loopX` devices), which is not recommended for production. To configure Docker to use a dedicated device for Device Mapper storage driver, an unformatted device or a partition (/dev/sdd) can be provided during installation. This directory is automatically configured and set up for use.
+Unless specified, Docker configuration defaults to the use of the overlay driver, which is not recommended for production.
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/prereq-hardware.adoc]: It IS recommended for production

File: modules/ROOT/pages/prereq-hardware.adoc [hunk 2]
```diff
@@ -148,7 +125,8 @@ NAME    MAJ:MIN RM  SIZE  RO TYPE MOUNTPOINT
 xvda    202:0    0   90G   0 disk
 ├─xvda1 202:1    0    1M   0 part
 └─xvda2 202:2    0   90G   0 part /
-xvdb    202:16   0  100G   0 disk {Docker volume (not formatted)}
+xvdb    202:16   0  100G   0 disk 
+/var/lib/gravity/planet/docker
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/prereq-hardware.adoc]: No new line

**Review by alejandrorascovan — DISMISSED** [review]

File: modules/ROOT/pages/prereq-hardware.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-hardware.adoc]: Got it - thanks!

File: modules/ROOT/pages/prereq-hardware.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-hardware.adoc]: Got it - thanks!

File: modules/ROOT/pages/prereq-hardware.adoc [hunk 3]
```diff
@@ -77,7 +77,14 @@ The main purpose of the etcd device is to provide dedicated storage for a distri
 
 Format this device either as `xfs` or `ext4` and mounted as `/var/lib/gravity/planet/etcd`. 
 
-The following shell snippet is provided as an example for mounting the etcd device using systemd mount files, and cannot be used as-is in production. The end result of this step mounts the Etcd device to the path `/var/lib/gravity/planet/etcd`. You can also mount the etcd device using `/etc/fstab` or some other method. Consult with your system administrator to determine the best way to meet this requirement. If you use the following example to guide this process, make sure to specify the correct device name in two places.
+The following shell snippet mounts the etcd device using systemd mount files to the path `/var/lib/gravity/planet/etcd`.
```
Comment (dhirsch1) [inline modules/ROOT/pages/prereq-hardware.adoc:80]: is "systemd" a typo or a real thing?

File: modules/ROOT/pages/prereq-hardware.adoc [hunk 4]
```diff
@@ -90,52 +97,24 @@ sudo systemctl start var-lib-gravity-planet-etcd.mount
...
-|202 |Xen virtual block devices (Amazon EC2)
-|===
-
-=== Reset Docker Device
-
-Using Logical Volume Manager, you can group multiple physical volumes into a single storage volume (volume group) and then divide these into logical volumes. Physical volumes are either a whole device or a partition.
-
-When a Docker device is in use by another logical volume or you want to manually reset a Docker device previously configured for Device Mapper, perform the following commands to reset the Docker device:
+[NOTE]
+Mount this device AFTER you mount the System State device under `/var/lib/gravity/planet`. If you are using system.d units to mount the volumes, declare mount dependencies using `After=<device-name>.<mount>`
```
Comment (dhirsch1) [inline modules/ROOT/pages/prereq-hardware.adoc:110]: Should "system.d" be "systemd", which is what you have other text?

**Review by dhirsch1 — APPROVED** [review]
Approved with one possible item to address

File: modules/ROOT/pages/prereq-hardware.adoc (hunk 4 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-hardware.adoc:110]: Yes - good catch!

**Pull request merged**